chapter/List-Of-Plugins.md

```diff
@@ -1,30 +1,29 @@
 ## 插件列表
 
-#### [以功能区分](#by-topic-1)
+#### [以功能区分](#以功能区分-1)
 
 - [文本对齐](#文本对齐)
-- [Building and linting](#building-and-linting)
-- [Code completion](#code-completion)
+- [构建和可疑代码标记](#构建和可疑代码标记)
+- [代码补全](#代码补全)
 - [Cycle](#cycle)
```

Comment:
Hmm, not really. Toggling means switching between only 2 states (binary, on/off), whereas "cycling" means switching between multiple states, e.g. buffers.